hikari/impl/shard.py

```diff
@@ -149,9 +149,9 @@ async def _receive_and_check(self, timeout: typing.Optional[float], /) -> str:
                 self._logger.error("connection closed with code %s (%s)", close_code, reason)
 
                 can_reconnect = close_code < 4000 or close_code in (
+                    errors.ShardCloseCode.UNKNOWN_ERROR,
```

Comment:
Do we want to die on an unknown error? It could be a one-off?

Reply:
Oh wait, misunderstood.

Reply:
This was already here. I just moved it up for it to have some sort of order. (4000, 4001, 4002, etc)